src/managing-workflow/configuring-dns.md

```diff
@@ -1,67 +1,71 @@
 # Configure DNS
 
-For Deis clusters, DNS records must be created. The Deis cluster runs multiple routers infront of the Deis controller and apps you deploy, so a [load balancer][] is recommended.
+The Deis Workflow controller and all applications deployed via Workflow are intended (by default) to be accessible as subdomains of the Workflow cluster's domain.  For example, assuming `example.com` were a cluster's domain:
 
-You can find the IP addresses of your Kubernetes cluster nodes by
-running `kubectl get nodes`.
+* The controller should be accessible at `deis.example.com`
 
```

Comment:
Yup... it looked nicer to me as I was editing the markdown, but you're right that it doesn't belong.  Removing it.